index.js

```diff
@@ -2,17 +2,15 @@
 
 var opal = require('opalcss')
 var extend = require('extend-shallow')
-var Promise = require('promise')
 
 exports.name = 'opalcss'
-exports.inputFormats = ['opal', 'opalcss']
+exports.inputFormats = ['opal', 'opalcss', 'css', 'scss', 'sass']
```

Comment:
RE: `inputFormats`... Not sure if Opal can process SCSS or SASS properly. Don't think it needs to process on already existing `.css` either.